### PR TITLE
Established connection to the Nordigen's api

### LINKED ISSUE #9: Basic connection to Nordigen Open Banking API
Create some script / simple app that connects to the API and pulls some data.

Figure out the output format.

### PR DESCRIPTION
**Pull request by Semerkosa**
Created a method which makes an authorized get request to the api and prints some data in JSON format.
Closes #9

### CODE REVIEW

**Review by stspot — APPROVED** [review]
The code looks good.

**Review by ANeshkova — APPROVED** [review]

**Review by bnenov — APPROVED** [review]

**Review by GGerginov — APPROVED** [review]

**Review by smanilov — CHANGES_REQUESTED** [review]
Please move to independent root directory.

File: ApiDemos/ApiDemos.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (smanilov) [inline ApiDemos/ApiDemos.iml:1]: It's unclear to me: what is this file for?

File: ApiDemos/src/NordigenApi.java [hunk 2]
```diff
@@ -1,20 +1,12 @@
...
 import java.net.http.HttpClient;
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
 
-@Component
-public class NordigenApiConnection implements CommandLineRunner {
-
-    @Override
-    public void run(String... args) throws Exception {
+public class NordigenApi {
```
Comment (smanilov) [inline ApiDemos/src/NordigenApi.java:8]: Just a note: the code still looks okay.

**Review by smanilov — CHANGES_REQUESTED** [review]
Please remove the .idea folder: everything in it is related to the local IDE configuration. (Unless it is needed in order to import the project in IntelliJ.)

Other than that: LGTM

File: ApiDemos/ApiDemos.iml (hunk 1 above)
Comment (Semerkosa) [reply ApiDemos/ApiDemos.iml:1]: Normally, it is used for keeping module configuration(created by default). However, it is unnecessary for the idea of the demo folder.

**Pull request merged**